src/main/java/io/github/nscuro/versatile/VersUtils.java

```diff
@@ -117,6 +120,53 @@ public static Vers versFromOsvRange(final String type, final String ecosystem, f
         return versBuilder.build();
     }
 
+    /**
+     * Convert a cpeMatch or exact version as used by NVD to a {@link Vers} range.
+     *
+     * @param cpeMatch      CpeMatch object for the CVE
+     * @param exactVersion  The exact version in CpeMatch
+     * @return The resulting {@link Vers}
+     * @throws IllegalArgumentException When the provided cpe match is invalid,
+     *                                  or the provided {@code events} contains an invalid event
+     * @throws VersException            When the produced {@link Vers} is invalid
+     * @throws InvalidVersionException  When any version in the range is invalid according to the inferred {@link VersioningScheme}
+     */
+    public static Vers versFromNvdRange(final CpeMatch cpeMatch, final String exactVersion) {
```

Comment:
Can we pass `version`, `versionStartExcluding`, `versionStartIncluding` etc. directly? I'd like to avoid adding `open-vulnerability-clients` as non-`test` dependency.

Reply:
Or, alternatively, create a custom `CpeMatch` that holds these fields.

Reply:
Yes, separate version variables would be better as apart from 4 versions, we're not using any information.